tests/unit/consensus/test_decisions_terminal.py

```
@@ -0,0 +1,456 @@
+            contract_snapshot_dict={"state": {}},
+            consensus_data_dict={"votes": {}, "leader_receipt": None, "validators": []},
+            timestamp_appeal=None,
+            leader_receipt=None,
+            validators=[],
+            redacted_consensus_data={"votes": {}},
+        )
+        defaults.update(overrides)
+        return defaults
+
+    def test_normal_undetermined_round(self):
+        effects, round_ = decide_undetermined(**self._base_kwargs())
```

Ruff (0.15.2)

[warning] Unpacked variable `effects` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)